src/core/pointcloud/qgspointcloudlayer.h

```diff
@@ -84,6 +85,24 @@ class CORE_EXPORT QgsPointCloudLayer : public QgsMapLayer, public QgsAbstractPro
        * Set to TRUE if point cloud index generation should be skipped.
        */
       bool skipIndexGeneration = false;
+
+      /**
+       * Set to true if the statistics calculation for this point cloud is disabled
+       * \since QGIS 3.26
+       */
+      bool skipStatisticsCalculation = false;
+    };
+
+
+    /**
+     * Point cloud statistics calculation task
+     * \since QGIS 3.26
+     */
+    enum PointCloudStatisticsCalculationState
```

Comment:
Enum class

Reply:
This change was actually in #48321
It will be addressed in an upcoming PR along with some other fixes.